packages/performance/wafer-map/src/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Wafer Map Performance Tests</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <meta name="description" content="Wafer map performance tests">
```

Comment:
This page will be published and linked from stroybook so iit should have the basics of a good html page.